### ISSUE TITLE
Error pyperclip

### ISSUE THREAD

**Issue by BenjaminMinguy** [idoavrah/terraform-tui#60]
Hello,

i have an error when i try to copy texte : 

```
╭──────────────────────────────────────────────── Traceback (most recent call last) ────────────────────────────────────────────────╮
│ /home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/tftui/__main__.py:467 in action_copy                      │
│                                                                                                                                   │
│   464 │                                                                                                                           │
│   465 │   def action_copy(self) -> None:                                                                                          │
│   466 │   │   if self.switcher.current == "resource":                                                                             │
│ ❱ 467 │   │   │   pyperclip.copy(self.app.tree.current_node.data.contents)                                                        │
│   468 │   │   │   self.notify("Copied resource definition to clipboard")                                                          │
│   469 │   │   elif self.switcher.current == "tree":                                                                               │
│   470 │   │   │   pyperclip.copy(self.app.tree.current_node.label.plain)                                                          │
│                                                                                                                                   │
│ ╭────────────────────────────────── locals ──────────────────────────────────╮                                                    │
│ │ self = TerraformTUI(title='Terraform TUI v0.12.5', classes={'-dark-mode'}) │                                                    │
│ ╰────────────────────────────────────────────────────────────────────────────╯                                                    │
│                                                                                                                                   │
│ /home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/pyperclip/__init__.py:659 in lazy_load_stub_copy          │
│                                                                                                                                   │
│   656 │   '''                                                                                                                     │
│   657 │   global copy, paste                                                                                                      │
│   658 │   copy, paste = determine_clipboard()                                                                                     │
│ ❱ 659 │   return copy(text)                                                                                                       │
│   660                                                                                                                             │
│   661                                                                                                                             │
│   662 def lazy_load_stub_paste():                                                                                                 │
│                                                                                                                                   │
│ ╭──────────────────────────────────────────── locals ────────────────────────────────────────────╮                                │
│ │ text = 'resource "aws_volume_attachment" "XXX" {\n    device_name = "XXX'+122 │                                │
│ ╰────────────────────────────────────────────────────────────────────────────────────────────────╯                                │
│                                                                                                                                   │
│ /home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/pyperclip/__init__.py:336 in __call__                     │
│                                                                                                                                   │
│   333 │   class ClipboardUnavailable(object):                                                                                     │
│   334 │   │                                                                                                                       │
│   335 │   │   def __call__(self, *args, **kwargs):                                                                                │
│ ❱ 336 │   │   │   raise PyperclipException(EXCEPT_MSG)                                                                            │
│   337 │   │                                                                                                                       │
│   338 │   │   if PY2:                                                                                                             │
│   339 │   │   │   def __nonzero__(self):                                                                                          │
│                                                                                                                                   │
│ ╭────────────────────────────────────────────── locals ───────────────────────────────────────────────╮                           │
│ │   args = ('resource "aws_volume_attachment" "XXX" {\n    device_name = XXX'+122,) │                           │
│ │ kwargs = {}                                                                                         │                           │
│ │   self = <pyperclip.init_no_clipboard.<locals>.ClipboardUnavailable object at 0x7f26f5f40cd0>       │                           │
│ ╰─────────────────────────────────────────────────────────────────────────────────────────────────────╯                           │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
PyperclipException:
    Pyperclip could not find a copy/paste mechanism for your system.
    For more information, please visit https://pyperclip.readthedocs.io/en/latest/index.html#not-implemented-error

For questions and suggestions, please visit https://github.com/idoavrah/terraform-tui/discussions
For issues and bugs, please visit https://github.com/idoavrah/terraform-tui/issues
```

Can you help me ?

**Comment by idoavrah** [comment]
From the ```pyperclip``` docs, regarding Linux support:

On Linux, this module makes use of the xclip or xsel commands, which should come with the os. Otherwise run “sudo apt-get install xclip” or “sudo apt-get install xsel” (Note: xsel does not always seem to work.)

I suggest you try installing either xclip or xsel. Please let me know if it worked.

I'll fix the application crash in a future release. Thanks!

**Comment by BenjaminMinguy** [comment]
I already installed xclip (and xsel too) but it is not working. I would like to copy some informations ( for terraform input for exemple) and copy or right click in windows does not working.

But it is a good tools, thanks very much.

Thanks for your feedback

**Comment by idoavrah** [comment]
Can you elaborate on your setup? Is it a Linux box? Some kind of a remote shell?

**Comment by BenjaminMinguy** [comment]
Yes, i am on linux ubuntu 20.04, shell bash, it is a virtual machine.

**Comment by idoavrah** [comment]
Sadly, it turns out it's not feasible. 
See https://stackoverflow.com/questions/51662213/cant-get-pyperclip-to-use-copy-and-paste-modules-on-python3

I managed to duplicate the issue, running TFTUI on my raspberrypi. Same issue :/

**Comment by BenjaminMinguy** [comment]
hum, ok, then i should upgrade my workstation... :'(

However, would it be possible to allow selection with the cursor to copy and paste a selection in a future version?

**Comment by idoavrah** [comment]
You can select parts of the text while holding a modifier keys (depends on the terminal and o/s):
- iTerm Hold the OPTION key.
- Gnome Terminal Hold the SHIFT key.
- Windows Terminal Hold the SHIFT key. 

Selecting multiple lines can introduce some extra characters. I'll consider adding a "full screen" toggle the help with this issue. Hope this helps.

**Comment by BenjaminMinguy** [comment]
Yeah it's Work, thanks very much

**Comment by idoavrah** [comment]
Fixed in [v0.12.6](https://github.com/idoavrah/terraform-tui/releases/tag/v0.12.6)
